src/main/java/com/ase/restservice/model/Asset.java

```diff
@@ -0,0 +1,30 @@
+package com.ase.restservice.model;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+@Entity
+@Table(name="asset")
+public class Asset {
+    @EmbeddedId
+    private AssetId assetId;
+    private Float numShares;
+    public Asset() {
+    }
+    public Asset(String accountId, String stockId, Float numShares) {
+        this.assetId = new AssetId(accountId, stockId);
+        this.numShares = numShares;
+    }
+    public String getAccountId() {
+        return assetId.getAccountId();
+    }
+    public String getStockId() { return assetId.getStockId();}
+
+    @Column(name = "numShares", nullable = false)
```

Comment:
Can we rename this "numShares" to "num_shares" since the SQL database is not case sensitive?